testnet/stacks-node/src/config.rs

```diff
@@ -2652,6 +2671,14 @@ impl NodeConfigFile {
                 .iter()
                 .filter_map(|contract_id| QualifiedContractIdentifier::parse(contract_id).ok())
                 .collect(),
+            fault_injection_block_push_fail_probability: if self
+                .fault_injection_block_push_fail_probability
+                .is_some()
+            {
+                self.fault_injection_block_push_fail_probability
+            } else {
+                default_node_config.fault_injection_block_push_fail_probability
+            },
```

Comment:
Also related to my other comment around using this: if this is field is for tests, do we even need to have it in `NodeConfigFile`?

Reply:
It's used not only for tests, but also for multi-miner testing.  So, it needs to be in the config file.